kernel/sys_clock.c

```diff
@@ -36,6 +36,8 @@ int z_clock_hw_cycles_per_sec;
  */
 static volatile u64_t tick_count;
 
+u64_t z_last_tick_announced;
```

Comment:
Shouldn't we protect access to this variable?

Reply:
This one is internal to the kernel only, so maybe not.  But it's going away anyway -- it exists as a temporary shim while the APIs switch around.